wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/en.md

```diff
@@ -33,29 +33,29 @@ Suggestions themselves should ideally consist of four main parts, though based o
 - **There is no such things as a "wrong" opinion** - it's up to the mapper and modders to come to an agreement in a civil manner - try to understand alternative points of view rather than attack them, as you never know when it might be useful in the future.
 - If you can't explain something through logic, you will not be able to explain it to them through violence either. Rather than continue a discussion that isn't progressing, walk away and try again with the next modder/mapper.
-- When things look to be getting out of hand, or of the discussion is struggling to stay on topic, [call a moderator](/wiki/Reporting_Bad_Behaviour) - don't contribute to the problem by continuing to post.
+- When things look to be getting out of hand, or if the discussion is struggling to stay on topic, [call a moderator](/wiki/Reporting_Bad_Behaviour). Don't contribute to the problem by continuing to post.
 
-## Qualified Mapsets
+## Qualified beatmaps
 
-The [Qualified section](/wiki/Beatmap/Category#qualified) allows mapsets to receive any final suggestions for change or improvement before reaching Ranked status. When modding a Qualified map, everything you consider "not minor" should **always** be mentioned - minor issues are also worth mentioning however, especially if they affect gameplay. If a Qualified mapset accumulates a significant number of minor issues, or if there is one or more major / unrankable issues, the mapset should be disqualified so that fixes and improvements can be made before attempting requalification.
+The [Qualified section](/wiki/Beatmap/Category#qualified) allows beatmaps to receive any final suggestions for change or improvement before reaching ranked status. When modding a qualified map, everything you consider "not minor" should **always** be mentioned—minor issues are also worth mentioning however, especially if they affect gameplay. If a qualified beatmap accumulates a significant number of minor issues, or if there is one or more major/unrankable issues, the beatmap should be disqualified so that fixes and improvements can be made before attempting requalification.
 
-If you're unsure on the nature of a potential problem, ask for other opinions or review previous mod posts to make sure the "issue" presented isn't something done intentionally by the mapper - if so, it will likely have been discussed already. If there has been no prior discussion, feel free to open one as if you were modding the map normally. If you feel the issues discovered should prevent the mapset from being ranked, report it in this thread with a link to your mod post and a brief outline of your concerns.
```

Comment:
was this sentence removed intentionally? feels somehow important to have it there

Reply:
there are three sentences: 1) ask around if you are unsure, 2) open a discussion if the issue wasn't mentioned, and 3) report the issue if it's important

I've asked in #nat, and it turned out the thread was closed in favour of beatmap discussion, which means that 2) and 3) both lead to the beatmap discussion area and eventually cover the same flow. so, they could be replaced with a single sentence instead of the two saying the same thing